src/main/java/fm/last/moji/impl/FileLengthCommand.java

```diff
@@ -102,6 +102,12 @@ private long getContentLength(HttpURLConnection httpConnection) {
     long length = 0;
     String rawLength = httpConnection.getHeaderField("Content-Length");
 
+    // Some webdav server, e.g., apache, does not return content-length header when the file size is zero
+    if(rawLength == null) {
```

Comment:
I guess the only issue could be what if there is no length header for some other reason, wouldn't it be better to return -1 indicating that the length couldn't be determined? I guess it's a bit of an edge case so if this works for you then it's probably fine. What are you doing with zero length files in the first place?